build.gradle

```diff
@@ -74,14 +74,17 @@ dependencies {
   }
 
   api 'com.segment.analytics.android:analytics:4.3.1'
-  api 'com.comscore:android-analytics:6.0.0'
+  api 'com.comscore:android-analytics:6.5.+'
```

Comment:
I think we should probably pin to a specific version to avoid breaking changes for customers. Partners do not always follow semver.